package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "deps": "python3 -m pip install -t lib -r requirements.txt --upgrade; cd py2; python3 -m pip install -t lib -r requirements.txt --upgrade",
-    "dev-deps": "python3 -m pip install --no-deps -r requirements.dev.txt",
+    "dev-deps": "python2 -m pip install --no-deps -r requirements.dev.txt",
```

Comment:
Do we need this for both py2 and py3?  I am not sure.

Reply:
My guess is it should be py2 only. I have removed `npm run dev-deps` from python3 CI and lets see if the tests pass

Reply:
Looks like you were right.